using-timescaledb/update-db.md

```diff
@@ -34,8 +34,11 @@ triggering the load of a previous TimescaleDB version on session startup.
 It must also be the first command you execute in the session. 
 <!-- -->
 
->:WARNING: If upgrading from a TimescaleDB version older than 0.12.0,
+>:WARNING: When upgrading from an old version of TimescaleDB before upgrading 
+to version 0.12.0 or version 1.5.0,
 you will need to restart your database before calling `ALTER EXTENSION`.
+After you upgraded to 1.6.1 you will need to restart the database 
```

Comment:
```suggestion
After upgrading to 1.6.1 you will need to restart the database 
```

Reply:
Dunno if this should be in a separate warning.  Also, maybe these should just be bolded notes for folks updating from particular versions, rather than warnings?

Reply:
It is already in the release notes for corresponding release. However, the general note was only for 0.12. I guess some users update through several versions at once, so they might not check notes for each version.